maas/plugins/horizon_check.py

```diff
@@ -111,5 +111,8 @@ def main(args):
         parser.add_argument('ip',
                             type=ipaddr.IPv4Address,
                             help='horizon dashboard IP address')
+        parser.add_argument('site_name_regexp',
+                            type=str,
+                            help='Horizon Site Name')
```

Comment:
@ameserole that's the default that will get used if someone is using our templates to configure checks/alarms. Not if someone is using the plugin directly. We used to hardcode the name inside the plugin, and we have removed that now. We should add it back in as a default to this new plugin arg.